package.json

```diff
@@ -116,8 +119,11 @@
     "sinon": "^3.2.1",
     "size-limit": "^0.11.0",
     "style-loader": "^0.18.2",
+    "tslint": "^5.9.1",
```

Comment:
Please add tslint to lint-staged config

Reply:
this is done, I'll be testing the PR on a project and update it as soon as I find any issue